doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-lanes-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='bpmn-lanes-proc_{context}']
+= BPMN2 swimlanes in process designer
+
+Swimlanes are process elements that visually group tasks related to one group or user. You can use user tasks in combination with swimlanes to assign multiple user tasks to the same actor, due to `Autoclaim` property of the swimlanes. When the first task in a swimlane is created with a specified actorId, then the same actorId is also assigned to all other tasks of that swimlane. The `Autoclaim` property enables the auto-assignment of the tasks that are related to a swimlane.
+
+NOTE: The assignment of actorId works when there is a single actor specified. However, if actorId contains multiple values, then the user tasks are not assigned automatically.
```

Comment:
I dont think necessary this note. As I mentioned before, this functionality targets user tasks with GroupId declared where a user must "claim" the task from that group and not tasks where ActorId is already declared (even though it may override in this case, but I would say this is a side effect and not the intend of the functionality).

Reply:
Maybe you can keep in the note only: _NOTE: If the remaining tasks contain 
multiple ActorIds predefined, then the user tasks are not assigned automatically_.

Reply:
Done, changed to:

NOTE: If the remaining user tasks in a swimlane contain multiple predefined `ActorIds`, then the user tasks are not assigned automatically.